packages/calcite-design-tokens/src/core/font.json

```diff
@@ -10,9 +10,44 @@
           "value": "Avenir Next World",
           "type": "fontFamilies"
         },
+        "avenir": {
+          "value": "Avenir",
+          "type": "fontFamilies",
+          "description": "Fallback for avenirNext"
```

Comment:
Nitpick: can the descriptions use the proper font name (e.g., `Avenir Next` vs `avenirNext`, `Monaco` vs `monaco`)?